src/app/execution-context.js

```diff
@@ -16,7 +16,10 @@ if (typeof window.web3 !== 'undefined') {
   web3 = new Web3(new Web3.providers.HttpProvider('http://localhost:8545'))
 }
 
-var vm = new EthJSVM(null, null, { activatePrecompiles: true, enableHomestead: true })
```

Comment:
Why did you removed the first two parameters? I might be one the reasons why tests are failing